host/testdata/xdc_integration_es_clusters.yaml

```diff
@@ -38,14 +33,9 @@
   clustermetadata:
     enableGlobalNamespace: true
     failoverVersionIncrement: 10
-    masterClusterName: "active-es"
+    masterClusterName: "standby-es"
```

Comment:
so both are master cluster?

Reply:
it doesn't matter here. I just try to mimic connecting two isolated clusters